### DISCUSSION TITLE
[Icon request] Bitcoin icon

### DISCUSSION THREAD

**Discussion by ibrunotome** [tailwindlabs/heroicons#419] [Ideas]

**Comment by ademers** [comment]
Would love to see a bitcoin icon! Or, perhaps a generic cryptocurrency icon. Would make a great addition to the current fiat currency icons. Thanks!

**Comment by mauriciabad** [comment]
Hey, I have a proposal!
The name of the icon would be: `currency-bitcoin`
(I haven't optimized the svg's)
The icon is very similar to the one I proposed here https://github.com/tailwindlabs/heroicons/discussions/493.

## 22x22
![currency-bitcoin-22](https://user-images.githubusercontent.com/12821361/148468352-b857d998-6a62-4160-90da-be7b7b8845a2.png)![currency-bitcoin-22-white](https://user-images.githubusercontent.com/12821361/148468532-4c9910bc-d762-4c11-823b-c1abfe46628a.png)
<img width="138" alt="Screenshot 2022-01-07 at 00 53 09" src="https://user-images.githubusercontent.com/12821361/148468781-e09aeccc-fd94-491c-91b3-c6a3326c119d.png">

```html
<svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10 8V12V16M10 8H11C12.933 8 14.5 8.20753 14.5 10C14.5 11.7925 12.933 12 11 12C12.933 12 15 12 15 14C15 16 12.933 16 11 16H10M10 8H9M10 16H9M12.5 7V7.5M13 17V16.5M10.5 17V16.5M10 7V7.5"/>
  <circle cx="12" cy="12" r="9" stroke="black" stroke-width="2"/>
</svg>
```

## 20x20
![currency-bitcoin-20](https://user-images.githubusercontent.com/12821361/148468355-dc60a6a0-df83-4e16-a1d9-3ec3e8a7d626.png)![currency-bitcoin-20-white](https://user-images.githubusercontent.com/12821361/148468544-bc87db85-18e9-46f7-a814-c29c3304fa03.png)
<img width="117" alt="Screenshot 2022-01-07 at 00 52 48" src="https://user-images.githubusercontent.com/12821361/148468802-2c5cba50-3703-46bd-a6d3-30e579d7b2c8.png">

```html
<svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" viewBox="0 0 20 20" fill="currentColor">
  <path fill-rule="evenodd" d="M10 18C14.4183 18 18 14.4183 18 10C18 5.58172 14.4183 2 10 2C5.58172 2 2 5.58172 2 10C2 14.4183 5.58172 18 10 18ZM9 5C9 4.44772 8.55228 4 8 4C7.44772 4 7 4.44772 7 5C6.44772 5 6 5.44772 6 6C6 6.55228 6.44772 7 7 7V10V13C6.44772 13 6 13.4477 6 14C6 14.5523 6.44772 15 7 15H7.5C7.5 15.5523 7.94772 16 8.5 16C9.05228 16 9.5 15.5523 9.5 15V14.9989C9.66392 14.9977 9.83133 14.9952 10 14.9902V15C10 15.5523 10.4477 16 11 16C11.5523 16 12 15.5523 12 15V14.7121C12.3739 14.5927 12.742 14.4167 13.0626 14.1516C13.6833 13.6385 14 12.9 14 12C14 11.3196 13.8202 10.7119 13.4291 10.2208C13.2869 10.0422 13.1269 9.8919 12.9565 9.76502C12.9932 9.71698 13.0286 9.667 13.0625 9.61503C13.3685 9.14601 13.5 8.59563 13.5 8C13.5 7.11873 13.2084 6.37502 12.5904 5.85069C12.2649 5.57454 11.8874 5.39339 11.5 5.2722V5C11.5 4.44772 11.0523 4 10.5 4C9.94772 4 9.5 4.44772 9.5 5V5.00542C9.33316 5.00145 9.16602 5 9 5ZM9 9V7C9.52389 7 9.97895 7.01613 10.3659 7.06848C10.8871 7.139 11.1604 7.26034 11.2965 7.37579C11.3828 7.44904 11.5 7.58658 11.5 8C11.5 8.3006 11.4357 8.44834 11.3874 8.52236C11.3399 8.59522 11.2557 8.67688 11.0755 8.7554C10.6579 8.93734 9.98293 9 9 9ZM9 13V11C10.0202 11 10.8386 11.0108 11.4118 11.1997C11.6722 11.2856 11.7989 11.3843 11.8646 11.4667C11.9214 11.5381 12 11.6804 12 12C12 12.3942 11.8839 12.5311 11.7883 12.6101C11.6407 12.7322 11.3585 12.8511 10.8593 12.922C10.3238 12.9981 9.6996 13 9 13Z" clip-rule="evenodd"/>
</svg>
```

**Reply by rri507** [reply]
H

**Comment by zefir-git** [comment]
Bitcoin is a logo. You can find the correct brand logo on https://simpleicons.org/
